content/en/docs/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-24-3.md

```diff
@@ -0,0 +1,67 @@
+---
+title: v2.24.3 Armory Release 
+toc_hide: true
+version: 02.24.02
+description: >
+  Release notes for Armory Enterprise v2.24.3
+---
+
+Filler page to get 2.24.3 things reviewed. As usual, Cameron will generate the file when ready
+
+## AWS Lambda 
+
+<!--https://github.com/spinnaker-plugins/aws-lambda-deployment-plugin-spinnaker/commit/7f7391855057daa34f03bd40796572eab2c9f51a#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5 -->
+
+> These improvements require version 1.0.8 of the AWS Lambda Plugin in addition to Armory Enterprise 2.24.3.
+
+This release includes the following new features and improvements for the Lambda provider:
+
+* Improved cache performance including fixes to cache issues found in 2.24.2
+* New configuration properties that give you greater control over how Armory Enterprise behaves when connection or cache issues occur.
+
+Configure the following properties in your Operator manifest (`spinnakerservice.yml` by default). Note that all these properties are optional and use the default if omitted. 
```

Comment:
Is this typically where plugin configuration goes? I would think this goes alongside the actual plugin configuration (in orca in this case)

Reply:
For Operator-based installs, yes (unless they're using Kustomize). There's an `orca` block  under `profiles`. The individual service profile files are for Halyard.